examples/nfs/rook-nfs.yaml

```diff
@@ -3,12 +3,12 @@ apiVersion: ceph.rook.io/v1
 kind: CephNFS
 metadata:
   name: my-nfs
-  namespace: default
 spec:
   # For Ceph v15, the rados block is required. It is ignored for Ceph v16.
```

Comment:
maybe just link https://rook.io/docs/rook/v1.9/ceph-nfs-crd.html ? 

rook 1.9+ no longer need `rados:` pool config.
I think its better to only support nfs export for cpeh v16.2.7+ just like rook.